tiledb/sm/filesystem/azure.h

```diff
@@ -389,19 +388,40 @@ class Azure {
   /**  The target block size in a block list upload */
   uint64_t block_list_block_size_;
 
+  /** The maximum number of retries. */
+  int max_retries_;
+
   /** The minimum time to wait between retries. */
   std::chrono::milliseconds retry_delay_;
 
+  /** The maximum time to wait between retries. */
+  std::chrono::milliseconds max_retry_delay_;
+
   /** Whether or not to use block list upload. */
   bool use_block_list_upload_;
 
+  /** The Blob Storage endpoint to connect to. */
+  std::string blob_endpoint_;
+
+  /** The Blob Storage account name. */
+  std::string account_name_;
+
+  /** The Blob Storage account key. */
+  std::string account_key_;
+
+  /** SSL configuration. */
+  SSLConfig ssl_cfg_;
+
   /** Maps a blob URI to its block list upload state. */
   std::unordered_map<std::string, BlockListUploadState>
       block_list_upload_states_;
 
   /** Protects 'block_list_upload_states_'. */
   std::mutex block_list_upload_states_lock_;
 
+  /** Protects 'client()'. */
+  mutable std::mutex client_init_lock_;
```

Comment:
Rename to client_init_mtx_ to match other clients.

Reply:
Done.